docs/running.rst

```diff
@@ -170,3 +170,43 @@ For example, to force YANK to use the ``OpenCL`` platform:
 See the ``yank script`` command line docs for more information on the ``-o`` flag.
 
 .. note:: The ``CPU`` platform will automatically use all available cores/hyperthreads in serial mode, but in MPI mode, will use a single thread to avoid causing problems in queue-regulated parallel systems.  To control the number of threads yourself, set the ``OPENMM_NUM_THREADS`` environment variable to the desired number of threads.
+
+
+Extending Simulations
+=====================
+
+One common operation when running simulations is to collect additional samples from an already run simulation to get
+better statistics. Alternately, when running on shared resources, you may need to break up long simulations into smaller
+simulations run in series. YANK provides a way to run its simulations in this manner by extending its simulations.
+
+YANK's :doc:`YAML <yamlpages/yaml>` files have two main options that work together to extend simulations.
+In order to extend simulations, set the following options in a YAML fie:
+
+.. code-block:: yaml
+
+  number_of_iterations: <Integer>
+  extension_simulation: True
+
+First you set :ref:`yaml_options_number_of_iterations` to an integer number of iterations you wish to extend the
+simulation. If no simulation has been run yet, then one will be run for the number of iterations.
+Setting :ref:`yaml_options_extension_simulation` to ``True`` modifies the behavior of
+:ref:`yaml_options_number_of_iterations` to extend the simulation by the specified number, adding on to what is already
+on the file.
+
+One could optionally just increase :ref:`yaml_options_number_of_iterations`, but then you have to change
+the YAML file every time you want to extend the run. Setting :ref:`yaml_options_extension_simulation` allows you to run
+the same YAML file without modification to do the same thing.
```

Comment:
It's probably worth pointing out that if your combinatorial job times out and you resume with the extension flag, it's going to extend also the simulations that have been already completed.